src/Quartz.Tests.Unit/Impl/Triggers/SimpleTriggerImplTest.cs

```diff
@@ -0,0 +1,379 @@
+using NUnit.Framework;
+using Quartz.Impl.Triggers;
+using System;
+
+namespace Quartz.Tests.Unit.Impl.Triggers
+{
+    [TestFixture]
+    public class SimpleTriggerImplTest
+    {
+        private Random _random;
+
+        public SimpleTriggerImplTest()
+        {
+            _random = new Random();
```

Comment:
in a CI environment a seed for random might be good, but this way it will show instability though...